platform/vs/tests/breakout/test_breakout_cli.py

```diff
@@ -0,0 +1,166 @@
+        self.setup_db(dvs)
+
+        output_dict = {}
+        brkoutTbl = swsscommon.Table(self.cdb, "BREAKOUT_CFG")
+        brkout_entries = brkoutTbl.getKeys()
+        assert len(brkout_entries) == 32
+
+        for key in brkout_entries:
+            (status, fvs) = brkoutTbl.get(key)
+            assert(status == True)
+
+            brkout_mode = fvs[0][1]
+            output_dict[key] = brkout_mode
+        output = collections.OrderedDict(sorted(output_dict.items(), key=lambda t: t[0]))
+        expected_dict = \
+                {'Ethernet8': '1x100G[40G]', 'Ethernet0': '1x100G[40G]', 'Ethernet4': '1x100G[40G]', \
+                'Ethernet108': '1x100G[40G]', 'Ethernet100': '1x100G[40G]', 'Ethernet104': '1x100G[40G]', \
+                'Ethernet68': '1x100G[40G]', 'Ethernet96': '1x100G[40G]', 'Ethernet124': '1x100G[40G]', \
+                'Ethernet92': '1x100G[40G]', 'Ethernet120': '1x100G[40G]', 'Ethernet52': '1x100G[40G]', \
+                'Ethernet56': '1x100G[40G]', 'Ethernet76': '1x100G[40G]', 'Ethernet72': '1x100G[40G]', \
+                'Ethernet32': '1x100G[40G]', 'Ethernet16': '1x100G[40G]', 'Ethernet36': '1x100G[40G]', \
+                'Ethernet12': '1x100G[40G]', 'Ethernet28': '1x100G[40G]', 'Ethernet88': '1x100G[40G]', \
+                'Ethernet116': '1x100G[40G]', 'Ethernet80': '1x100G[40G]', 'Ethernet112': '1x100G[40G]', \
+                'Ethernet84': '1x100G[40G]', 'Ethernet48': '1x100G[40G]', 'Ethernet44': '1x100G[40G]', \
+                'Ethernet40': '1x100G[40G]', 'Ethernet64': '1x100G[40G]', 'Ethernet60': '1x100G[40G]', \
+                'Ethernet20': '1x100G[40G]', 'Ethernet24': '1x100G[40G]'}
+        expected = collections.OrderedDict(sorted(expected_dict.items(), key=lambda t: t[0]))
+        assert output == expected
+
+    # Breakout Cli Test Mode
+    def test_breakout_modes(self, dvs):
+        expected = self.read_Json(dvs)
+        assert expected
+
+        print("**** Breakout Cli test Starts ****")
+        output_dict = self.breakout(dvs, 'Ethernet0', '2x50G')
+        expected_dict = expected["Ethernet0_2x50G"]
+        assert output_dict == expected_dict
+        print("**** 1X100G --> 2x50G passed ****")
+
+        output_dict = self.breakout(dvs, 'Ethernet4', '4x25G[10G]')
+        expected_dict = expected["Ethernet4_4x25G"]
+        assert output_dict == expected_dict
+        print("**** 1X100G --> 4x25G[10G] passed ****")
+
+        output_dict = self.breakout(dvs, 'Ethernet8', '2x25G(2)+1x50G(2)')
+        expected_dict = expected["Ethernet8_2x25G_1x50G"]
+        assert output_dict == expected_dict
+        print("**** 1X100G --> 2x25G(2)+1x50G(2) passed ****")
+
+        output_dict = self.breakout(dvs, 'Ethernet12', '1x50G(2)+2x25G(2)')
+        expected_dict = expected["Ethernet12_1x50G_2x25G"]
+        assert output_dict == expected_dict
+        print("**** 1X100G --> 1x50G(2)+2x25G(2) passed ****")
+
+        #TODOFIX: remove comments once #4442 PR got merged.
```

Comment:
Why does this part depend on #4442 and the other test cases do not?

Reply:
As per DPB [fixure](https://github.com/Azure/sonic-swss/blob/master/tests/conftest.py#L1112), we have **"BREAKOUT_CFG", "DEVICE_METADATA" and "PORT"** keys in config_db.json. so after breakout things working out successfully(like what ports we have to add or delete), we need to set the current breakout mode to the target breakout mode. 
But to do that, we have to have all three yang models atlease i.e. "BREAKOUT_CFG", "DEVICE_METADATA" and "PORT". Though yang models are already available for "BREAKOUT_CFG" and "PORT" currently. Yang model for "DEVICE_METADATA" is still missing. 
#4442 PR has yang model for "DEVICE_METADATA" which we need if we want to transform from any other non-default breakout mode(not 100 G) to any other breakout mode.

That's why, the current test cases are all from default breakout mode(1x100G). I can remove the rest part as of now. Once #4442 PR gets merged, I can add these test cases. Let me know what you think.

Reply:
I think it's ok to leave this here, was just wondering! Maybe expand the comment a bit to mention that these test cases depend on DEVICE_METADATA to go from a non-default breakout mode to a different breakout mode. :)

Reply:
Addressed as requested.